src/main/java/com/conveyal/gtfs/TripPatternKey.java

```diff
@@ -99,6 +102,8 @@ public int hashCode() {
         result = 31 * result + (stops != null ? stops.hashCode() : 0);
         result = 31 * result + (pickupTypes != null ? pickupTypes.hashCode() : 0);
         result = 31 * result + (dropoffTypes != null ? dropoffTypes.hashCode() : 0);
+        result = 31 * result + (start_pickup_dropoff_window != null ? start_pickup_dropoff_window.hashCode() : 0);
```

Comment:
Does the 31 have to change now that we have added variables? Isn't there now a clash potential

Reply:
Possibility. I've updated the method to use the newer approach to this:

`Objects.hash(routeId, stops, pickupTypes, dropoffTypes, start_pickup_dropoff_window, end_pickup_dropoff_window);`